designing-reports/report-designer-tools/desktop-designers/standalone-report-designer/configuration/extending-report-designer.md

```diff
@@ -1,66 +1,63 @@
 ---
 title: Extending Report Designer
-page_title: Extending Report Designer 
-description: Extending Report Designer
+page_title: Extending Report Designer at a Glance
+description: "Learn how to make the Telerik Reporting Standalone Designer recognize custom CLR public methods and properties to be used as User functions and ObjectDataSources."
 slug: telerikreporting/designing-reports/report-designer-tools/desktop-designers/standalone-report-designer/configuration/extending-report-designer
 tags: extending,report,designer
 published: True
 position: 2
 previous_url: /standalone-report-designer-extending-configuration
 ---
 
-# Extending Report Designer
+# Extending Report Designer to Recognize Custom Objects
```

Comment:
Maybe use "Custom Assemblies" instead? Otherwise, it would feel like the article is about the objectdatasource only when it is also valid for user functions